game/configs/mission/cfg_equipment.hpp

```diff
@@ -3,29 +3,625 @@ class vgm_equipment {
         condition = "true";
 
         weapons[] = {
-            "hgun_P07_F"
+            "vn_m45",
+            "vn_m1carbine",
+            "vn_m1_garand",
+            "vn_xm177",
+            "vn_m1911",
+            "vn_p38",
+            "vn_m127"
         };
+
         magazines[] = {
-            "16Rnd_9x21_Mag"
+            // Weapon magazines
+            "vn_m45_mag",
+            "vn_m45_t_mag",
+            "vn_carbine_15_mag",
+            "vn_carbine_15_t_mag",
+            "vn_carbine_30_mag",
+            "vn_carbine_30_t_mag",
+            "vn_m1_garand_mag",
+            "vn_m1_garand_t_mag",
+            "vn_m16_20_mag",
+            "vn_m16_20_t_mag",
+            "vn_m1911_mag",
+            "vn_p38_mag",
+            // Flare launcher flares
+            "vn_m127_mag",
+            "vn_m128_mag",
+            "vn_m129_mag",
+            // Grenades and explosives
+            "vn_m18_green_mag",
+            "vn_m18_purple_mag",
+            "vn_m18_red_mag",
+            "vn_m18_white_mag",
+            "vn_m18_yellow_mag",
+            "vn_m61_grenade_mag",
+            "vn_mine_satchel_remote_02_mag",
+            "vn_mine_m14_mag"
         };
-        backpacks[] = {};
+
+        backpacks[] = {
+            "vn_b_pack_01",
+            "vn_b_pack_02",
+            "vn_b_pack_04",
+            "vn_b_pack_05"
+        };
+
         items[] = {
-            "U_B_CombatUniform_mcam_tshirt"
+            // Accessories
+            "vn_s_ppk",
+            "vn_s_m1911",
+            // Uniforms
+            "vn_b_uniform_sog_01_01",
+            "vn_b_uniform_sog_01_02",
+            "vn_b_uniform_sog_01_03",
+            "vn_b_uniform_sog_01_04",
+            "vn_b_uniform_sog_01_05",
+            "vn_b_uniform_sog_01_06",
+            "vn_b_uniform_sog_02_01",
+            "vn_b_uniform_sog_02_02",
+            "vn_b_uniform_sog_02_03",
+            "vn_b_uniform_sog_02_04",
+            "vn_b_uniform_sog_02_05",
+            "vn_b_uniform_sog_02_06",
+            // Vests
+            "vn_b_vest_sog_01",
+            "vn_b_vest_sog_02",
+            "vn_b_vest_sog_03",
+            "vn_b_vest_sog_04",
+            "vn_b_vest_sog_05",
+            "vn_b_vest_sog_06",
+            // Headgear
+            "vn_b_headband_01",
+            "vn_b_headband_02",
+            "vn_b_headband_03",
+            "vn_b_headband_04",
+            "vn_b_headband_08",
+            // Equipment
+            "vn_camera_01",
+            "vn_anpvs2_binoc",
+            "vn_m19_binocs_grey",
+            "vn_m19_binocs_green",
+            "vn_mk21_binocs",
+            "vn_b_item_map",
+            "vn_b_item_radio_urc10",
+            "vn_b_item_compass",
+            "vn_b_item_watch"
         };
     };
 
     class rifleman {
-        condition = "missionNamespace getVariable ['test', false]";
+        condition = "[vgm_skills_pathsHashMap get ['rifleman', 'loadout']] call vgm_g_fnc_skills_isKnown";
```

Comment:
Use API instead of getting it from the variable
```suggestion
        condition = "(['rifleman', 'loadout'] call vgm_g_fnc_skills_getByPath) call vgm_g_fnc_skills_isKnown";
```